backend/user-service/src/test/python/test.py

```
@@ -0,0 +1,27 @@
+import requests
+
+
+reg = {
+    "username": "stiboly",
+    "password": "xx123123123xx",
+    "email": "landric6124si@raf.rs",
+}
+res = requests.post("http://localhost:8081/api/v1/users", json=reg)
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)